src/components/SidebarLayout/Header/components/Layout.tsx

```diff
@@ -61,13 +62,13 @@ const Layout = openHoc(({ classes, clickAway, open, providerDetails, providerInf
   <Row className={classes.summary}>
     <Col className={classes.logo} middle="xs" start="xs">
       <Link to="/">
-        <Img alt="Gnosis Team Safe" height={32} src={logo} testId="heading-gnosis-logo" />
+        <Img alt="Gnosis Team Safe" height={36} src={logo} testId="heading-gnosis-logo" />
       </Link>
     </Col>
+    {/* <Divider />
+     <SafeListHeader />
     <Divider />
-    <SafeListHeader />
-    <Divider />
-    <NetworkLabel />
+    <NetworkLabel />  */}
```

Comment:
Isn't it safe to remove this comments?

Any future use note?

Reply:
removed